### DISCUSSION TITLE
USB hubs

### DISCUSSION THREAD

**Discussion by n19htmare** [master-hax/pixel-backup-gang#19] [General]
Thank you :) what hub is that in the picture?

**Comment by master-hax** [comment]
the USB C hub in the demo picture is from "QGeeM"

**Comment by Arora-Sir** [comment]
> Thank you :) what hub is that in the picture?

After conducting thorough research, I found that the Ambrane USB C hub also worked, so I purchased this one: `<redacted amazon referral link>`

**Comment by master-hax** [comment]
thank you - please try to avoid sharing affiliate / tracking links

**Comment by davidsarkany** [comment]
I bought a noname hub from aliexpress.
$5, usb-c, charging and ethernet works fine.
Lot of seller selling this hub, use this image for search the product. 
(You can search product by image with Aliexpress)

![8in1-USB-C-HUB-Type-C-Splitter-4K-Thunderbolt-3-Docking-Station-Laptop-Adapter-For-Macbook jpg_](https://github.com/user-attachments/assets/46ab5f14-ead1-4064-8bd3-ea9cc10d7302)
